bin/main.rb

```diff
@@ -0,0 +1,2 @@
+puts "Hello world!"
```

Comment:
main.rb should be an [executable](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable/) file, add `#!/usr/bin/env ruby` at the top